app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -31,10 +31,14 @@ ListingController = ($scope, $state, $sce, $sanitize, SharedService, ListingServ
   $scope.isActiveTable = (table) ->
     $scope["active#{table}Class"] == 'active'
 
-  $scope.unitAreaRange = (units) ->
-    # TODO: actually find min/max
-    # if units.length == 1
-    units[0].Unit_Square_Footage
+  $scope.unitAreaRange = (summary) ->
```

Comment:
i noticed that `summary` pertains to a listing summary -- but not sure what is a listing summary. can you clarify how it is different from just regular listing?